packages/storybook/package.json

```diff
@@ -5,7 +5,6 @@
   "scripts": {
     "dev": "storybook dev -p 6006",
     "build": "storybook build",
-    "preview-storybook": "serve storybook-static",
```

Comment:
This removal intentional?

Reply:
Yeah we don't particularly use it, and adds another dependency